CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 All notable changes to this project will be documented in this file.
 This project adheres to Semantic Versioning (http://semver.org/).
 
+### 7.0.0
+* Update category model
```

Comment:
Needs to be more descriptive, otherwise looks good

Reply:
Updated